internal/db/schema/migrations/postgres/11/01_server_type_enum.up.sql

```diff
@@ -0,0 +1,17 @@
+begin;
+
+create table server_type_enm (
+  name text primary key not null,
+  constraint only_predefined_server_types_allowed
+    check (name = ANY (array['controller'::text, 'worker'::text]))
+);
+
+insert into server_type_enm (name) values
+  ('controller'),
+  ('worker');
+
+alter table server
+    add foreign key (type) references server_type_enm(name)
```

Comment:
We name foreign key constraints. Can you add a name following our [foreign key](https://github.com/hashicorp/boundary/blob/c9e4c877b89038c9e9d4f1edecb15a1123073519/internal/db/schema/migrations/postgres/2/04_oidc.up.sql#L16-L17) [naming pattern](https://github.com/hashicorp/boundary/blob/c9e4c877b89038c9e9d4f1edecb15a1123073519/internal/db/schema/migrations/postgres/10/04_vault_credential.up.sql#L167-L168)?

Reply:
Fixed!